<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoordinatusWidget</class>
 <widget class="QWidget" name="CoordinatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_posX">
      <property name="styleSheet">
       <string notr="true">     border-width: 0px;
     border-style: solid;
     border-color: darkblue;</string>
      </property>
      <property name="text">
       <string>pos X axis</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="c_label_posX_value">
      <property name="styleSheet">
       <string notr="true">     border-width: 0px;
     border-style: solid;
     border-color: darkblue;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_posY">
      <property name="styleSheet">
       <string notr="true">     border-width: 0px;
     border-style: solid;
     border-color: darkblue;</string>
      </property>
      <property name="text">
       <string>pos Y axis</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="c_label_posY_value">
      <property name="styleSheet">
       <string notr="true">     border-width: 0px;
     border-style: solid;
     border-color: darkblue;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_posZ">
      <property name="styleSheet">
       <string notr="true">     border-width: 0px;
     border-style: solid;
     border-color: darkblue;</string>
      </property>
      <property name="text">
       <string>pos Z axis</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="c_label_posZ_value">
      <property name="styleSheet">
       <string notr="true">     border-width: 0px;
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_posE">
      <property name="styleSheet">
       <string notr="true">     border-width: 0px;
     border-style: solid;
     border-color: darkblue;</string>
      </property>
      <property name="text">
       <string>pos E axis</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="c_label_posE_value">
      <property name="styleSheet">
       <string notr="true">     border-width:0px;
     border-style: solid;
     border-color: darkblue;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="c_label_positioning_value">
      <property name="text">
       <string>Abs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_positioning">
      <property name="text">
       <string>Positioning</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_modelState">
      <property name="text">
       <string>modelState</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="c_label_modelState">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_queueState">
      <property name="text">
       <string>queueState</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="c_label_queueState">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_temperature">
      <property name="text">
       <string>temperature</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="c_label_temperature">
      <property name="text">
       <string>20</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="c_label_contiCnt">
      <property name="toolTip">
       <string>countinue counter</string>
      </property>
      <property name="text">
       <string>contiCnt</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_contiCnt">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
